doc/src/examples/algorithms/create_svg_overlay.hpp

```diff
@@ -45,10 +46,10 @@ void create_svg(std::string const& filename, Geometry const& a, Geometry const&
                     "fill:rgb(0,0,0);font-family:Arial;font-size:10px");
     }
 #else
-    boost::ignore_unused_variable_warning(filename);
-    boost::ignore_unused_variable_warning(a);
-    boost::ignore_unused_variable_warning(b);
-    boost::ignore_unused_variable_warning(range);
+    boost::ignore_unused(filename);
+    boost::ignore_unused(a);
+    boost::ignore_unused(b);
+    boost::ignore_unused(range);
```

Comment:
This could be:

        boost::ignore_unused(filename, a, b, range);

What is the intention in using one call per variable? E.g. do you think it is clearer this way?

Reply:
@awulkiew Good point. I've updated the commit.